src/dawn/Serialization/IIRSerializer.cpp

```diff
@@ -798,7 +798,7 @@ IIRSerializer::deserialize(const std::string& file, OptimizerContext* context,
 
   std::string str((std::istreambuf_iterator<char>(ifs)), std::istreambuf_iterator<char>());
   std::shared_ptr<iir::StencilInstantiation> returnvalue =
-      std::make_shared<iir::StencilInstantiation>(context);
+      std::make_shared<iir::StencilInstantiation>();
```

Comment:
we need to make sure that we all use the same clang-format version :-)

Reply:
the  context actually got removed here, something did change :)

Reply:
somehow the comment is at the wrong place...it's for line 646.